cmake/packaging.cmake

```diff
@@ -238,6 +238,8 @@ list(GET OSQUERY_VERSION_COMPONENTS 1 CPACK_PACKAGE_VERSION_MINOR)
 list(GET OSQUERY_VERSION_COMPONENTS 2 CPACK_PACKAGE_VERSION_PATCH)
 
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "osquery is an operating system instrumentation toolchain.")
+set(CPACK_PACKAGE_NAME "osquery")
+set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
```

Comment:
Any reason why these are explicitly specified? That's the default.

Reply:
I found that I had to explicitly set these for the line below: https://github.com/osquery/osquery/pull/5770/files#diff-bfc91d448f2fbd980743d90de3646bbbR259